services/Application.js

```diff
@@ -93,6 +94,29 @@ function stopApp(appName, deviceId, cb) {
   }, cb);
 }
 
+//Group support
+
+function startGroupApp(appName, groupName, cb) {
+  sendDeviceCommand('app-group-start', {
+    name: appName,
```

Comment:
the MXSS is ignorant of groups, it doesn't access firebase. please issue an app-start command for each application in the group

Reply:
this is old code that i missed to remove it. we are not using. We send app-start for each application. this was an option if it should handle with the worker. but it'sn't necessary